src/components/tooltip/_tooltip.scss

```diff
@@ -9,6 +9,7 @@
   display: inline-block;
   height: 100%;
   width: 100%;
+  z-index: 1;
```

Comment:
don't we have other elements with z-index 1 as well ? is any of them in a position where it can hide seekbar ?
i think overlays have z-index 1 as well

Reply:
not sure I understand the question.. this issue is a regression from this [PR](https://github.com/kaltura/playkit-js-ui/pull/782/files). since we increased the z-index of the seekbar area, it overlays the tooltip of the control buttons which is inside a sibling container of the bottom bar.